viewer/packages/camera-manager/src/CameraManagerHelper.ts

```diff
@@ -0,0 +1,164 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import * as THREE from 'three';
+import range from 'lodash/range';
+
+/**
+ * Helper methods for implementing a camera manager.
+ */
+export class CameraManagerHelper {
```

Comment:
Can this be unit tested without a ton of effort?

Reply:
fitCameraToBoundingBox test could be quite simple. Probably can think of some simple tests with near and far plane as well. The same probably applies for targetFromRotation, as an example we can take a 180° rotation and check whether new target is on corresponding position, with initial target being on one of the axes.

Reply:
Create a test for targetFromRotation